crates/tx5-go-pion-sys/build.rs

```diff
@@ -184,25 +184,87 @@ fn go_build() {
             .success(),
         "error running go build",
     );
+}
 
-    use sha2::Digest;
-    let data = std::fs::read(lib_path).expect("failed to read generated lib");
-    let mut hasher = sha2::Sha256::new();
-    hasher.update(data);
-    let hash =
-        base64::encode_config(hasher.finalize(), base64::URL_SAFE_NO_PAD);
-
-    let mut exe_hash = out_dir;
-    exe_hash.push("lib_hash.rs");
-    std::fs::write(
-        exe_hash,
-        format!(
-            r#"
-        const LIB_HASH: &str = "{hash}";
-    "#,
-        ),
-    )
-    .expect("failed to write lib hash");
+fn go_build() {
+    let out_dir = std::env::var("OUT_DIR")
+        .map(std::path::PathBuf::from)
+        .expect("error reading out dir");
+
+    let manifest_path = std::env::var("CARGO_MANIFEST_DIR")
+        .map(std::path::PathBuf::from)
+        .expect("error reading manifest dir");
+
+    let cp = |f: &'static str| {
+        let mut a = manifest_path.clone();
+        a.push(f);
+        let mut b = out_dir.clone();
+        b.push(f);
+        std::fs::copy(a, b).expect("failed to copy go file");
+    };
+
+    cp("buffer.go");
+    cp("const.go");
+    cp("datachannel.go");
+    cp("main.go");
+    cp("peerconnection.go");
+    cp("go.sum");
+    cp("go.mod");
+
+    match TARGET.link_type {
+        LinkType::Dynamic => {
+            let mut lib_path = out_dir.clone();
+
+            match TARGET.go_os {
+                Some("darwin") | Some("ios") => {
```

Comment:
Used to reference `"macos" | "ios" | "tvos"` where we now have translations to go_os: `macos` -> `darwin` and `ios` -> `ios`. Should `tvos` flow through here?

Also, do we need to support tvos? I can't think of a use-case for tx5 on a TV but I guess we may as well be consistent about building on the platforms we can :)

Reply:
These are now based on the [valid targets that go can build for](https://github.com/golang/go/blob/master/src/go/build/syslist.go#L14-L33) since that's all we can support anyways. They don't seem to differentiate tvos, I'm not sure if that's just included in ios or what.